src/pretix/presale/views/cart.py

```diff
@@ -56,13 +56,24 @@ def _items_from_post_data(self):
                 parts = parts[:-1]
             else:
                 voucher = None
+
+            try:
+                amount = int(value)
+                assert amount > 0
```

Comment:
Could we use a simple ``if`` instead of an ``assert``? ``assert`` is not run if ``__debug__`` is ``False`` which might happen in production (``python -O``).